gravitee-io/gravitee-access-management
# feat: manage i18n dictionary from the file system to adapt email temp…

…lates

fixes gravitee-io/issues#8039

# How to test it

* create a new user in the default idp
* enable forgot password on the application
* go to the app and request a new password (you should receive the EN version)
* in the console, update the user profile to add the `locale` addition information
![image](https://user-images.githubusercontent.com/3110552/179558675-570a0184-8f1b-4d2e-ade8-40fd23958929.png)
* request a new password, the email should be in FR
